Hackney.Shared.HousingSearch/Gateways/Models/Tenures/QueryableTemporaryAccommodationInfo.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+using Nest;
```

Comment:
Is this using required here?

Reply:
Nope, vestigial. Deleted in [df2a5bf](https://github.com/LBHackney-IT/housing-search-shared/pull/73/commits/df2a5bf616102dc6a7ca02142a1d1b7ae793d12d).